src/org/ods/orchestration/util/DocumentHistoryEntry.groovy

```diff
@@ -0,0 +1,148 @@
+package org.ods.orchestration.util
+
+import com.cloudbees.groovy.cps.NonCPS
+
+class DocumentHistoryEntry implements Map, Serializable {
+
+    private final Long entryId
+    private final Map delegate
+    private final String projectVersion
+    private final String previousProjectVersion
+    String rational
```

Comment:
this fix encapsulation just to allow unit test to be able to change the... a solution be to convert it to 'private String rational' and additionally this method needs to be added to avoid breaking the unit test:
```
@NonCPS
    String setRational(rational) {
        return this.rational = rational
    }
```

Reply:
not sure if that's needed @stitakis... in the end, Groovy automatically creates getters and setters for public fields, which is the case here. Unless we need a specific setter or getter that is off-standard, no need to create a longer codebase. agreed?